simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -295,10 +295,18 @@ function simple_ldap_user_import($form, &$form_state) {
   asort($users);
 
   // Only selects new users
-  $drupal_users = entity_load('user');
+  
   $new_names = $users;
-  foreach ($drupal_users as $user) {
-      unset($new_names[strtoupper($user->name)]);
+  $sql="select name from {users} where name= :name";
+  foreach($new_names as $item){
+     $result=db_query($sql, array(':name'=>$item));
+     if($result)
```

Comment:
Please adjust spacing above and below and move the opening brace onto the same line, per Drupal coding standards for control structures.

https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct